x/stakeibc/types/ica_account.go

```diff
@@ -1,5 +1,21 @@
 package types
 
-func FormatICAAccountOwner(chainId string, accountType ICAAccountType) (result string) {
+import fmt "fmt"
+
+// Helper function to build the host zone ICA owner in the form "{chainId}.{ICA_TYPE}"
+func FormatHostZoneICAOwner(chainId string, accountType ICAAccountType) (result string) {
 	return chainId + "." + accountType.String()
 }
+
+// Helper function to build the ICA owner for a trade route ICA
+// in the form "{chainId}.{rewardDenom}-{hostDenom}.{ICA_TYPE}"
+func FormatTradeRouteICAOwner(chainId, rewardDenom, hostDenom string, icaAccountType ICAAccountType) string {
```

Comment:
nit: add string as type for first two params for consistency

Reply:
this was done intentionally - it's a common practice in go when you have params of the same type

Reply:
ok, not my favorite style thing but I see how it works